src/constructs/core/identity.ts

```diff
@@ -0,0 +1,5 @@
+export interface Identity {
+  stack: string;
+  stage: string;
+  app: string;
+}
```

Comment:
Wasn't entirely sure where to put this so open to suggestions of where it could live.